src/lib/data/near.js

```diff
@@ -82,6 +135,27 @@ async function functionCall(
   try {
     const wallet = await (await near.selector).wallet();
 
+    window.near = near;
```

Comment:
I understand this is still in draft but would it be possible not to override `window.near` since it's used by Sender Wallet?

Reply:
yes, this was just for my own debugging, and will not be merged in with this PR. I will remove it!